packages/server/test/support/fixtures/projects/ts-proj-with-own-tsconfig/cypress/support/commands.ts

```diff
@@ -4,3 +4,10 @@
 Cypress.Commands.add('clickLink', (label: string | number | RegExp) => {
   cy.get('a').contains(label).click()
 })
+
+// https://github.com/cypress-io/cypress/issues/7510
+// The code below fails when @typescript-eslint/no-misused-promises is error
+// and the return type of function in Cypress.Commands.add doesn't support Chainable.
+Cypress.Commands.add('dataCy', (value) => {
```

Comment:
without the types change this test fails?

Reply:
tsc doesn't fail. typescript-eslint fails because of the rule [@typescript-eslint/no-misused-promises](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-misused-promises.md)